### PR TITLE
Make asciidoc admonitions more noticeable

### PR DESCRIPTION
**Pull request by rsandell**
Slightly mimics how it looks on GitHub.

Related to issue # 
Part of https://issues.jenkins-ci.org/browse/WEBSITE-690

Summary of this pull request: 

If the plugin documentation is migrated to asciidoc and admonitions is used (the note and info boxes) it doesn't stick out as it should on the plugin site.

This is what it will look like after the change:

pluginsite_admonition_change

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]
Looks right

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
Change looks fine, should it be the plugin's responsibility to add security warnings though? 
I would think it should be done through the plugin-site?

(doc in question: https://github.com/jenkinsci/gerrit-trigger-plugin/blob/master/docs/README.adoc)

**Comment by zbynek** [comment]
agreed with @timja , discussion on that topic can go to https://github.com/jenkinsci/gerrit-trigger-plugin/pull/405

Does not block this PR though.

**Comment by rsandell** [comment]
The security warning is just something that followed along with the wiki extractor tool, there are other info and note admonitions in the docs.

**Comment by rsandell** [comment]
So who should I ping to get this merged?

**Comment by daniel-beck** [comment]
> The security warning is just something that followed along with the wiki extractor tool

Right, should be removed though as the plugin site has that content in the sidebar (and otherwise was dynamically generated anyway).

**Comment by timja** [comment]
@daniel-beck @oleg-nenashev and @kohsuke are the maintainers according to the github team (I've requested to join via GitHub)

**Review by oleg-nenashev — APPROVED** [review]
Would rather automatically remove ones which are already supported by the plugin site, but fine as improvement

**Pull request merged**
